index.js

```diff
@@ -368,7 +379,20 @@ function fastifyView (fastify, opts, next) {
     })
   }
 
-  next()
+  if (prod && type === 'handlebars' && options.partials) {
```

Comment:
why this changes in the `mustache` handler?

Reply:
The only change is just to update the function name there.